### ISSUE TITLE
“NeuroGentoo”

### ISSUE THREAD

**Issue by TheChymera** [gentoo/sci#635]
For a while now, I have contributed quite a few of neuroscience ebuilds; these have already helped me and my colleagues easily set up a number of now very busy data analysis boxes. Somewhere along the line I realized that this has a lot of potential to help other researchers as well (and grow the Gentoo user base), so I started calling the whole thing NeuroGentoo and presenting it at meetings and hackathlons as an alternative to NeuroDebian (with the agreement of the Gentoo Trustees, ofc).

I am sure you are all familiar with the reasons one might prefer Gentoo over Debian as a base distro. 

While the name NeuroGentoo might be useful for highlighting the potential of a NeuroDebian-analogue solution under Gentoo, it isn't really descriptive of the actual state of affairs. NeuroGentoo is a subset of Gentoo Science, and I am becoming more and more convinced there are also advantages in presenting it as such: 

All the additional packages in Gentoo Science are undoubtedly valuable to neuroscientists in the ever-more-interdisciplinary world of research; and of course all the benefits of NeuroGentoo, which I like to emphasize, also extend to all the other packages.

So basically, I'm thinking of no longer talking to people about _NeuroGentoo_ but talking to them about _Gentoo Science_. I would feel a lot better while doing that if I knew you approved. To give you an idea of the context in which I am trying to push for NeuroGentoo (and would in the future push for Gentoo Science): [here](https://hbm_brainhack2016.hackpad.com/) is the abstract for the upcoming OHBM Open Science Special Interest Group.
